bbc/simorgh
# Use @bbc/psammead-amp-geo

Resolves https://github.com/bbc/psammead/issues/298

**Overall change:** Component `@bbc/psammead-amp-geo` has been created in PR: https://github.com/bbc/psammead/pull/1367 and is being used in Simorgh

**Code changes:**

- Uses `@bbc/psammead-amp-geo` 

**How to test this is working**

- Visit an amp page: http://localhost.bbc.com:7080/news/articles/c0g992jmmkko.amp In an EEA country, or using a VPN to access an EEA country (including the UK)
- Inspect source.  The `<body>` element should have the attributes `amp-geo-group-eea` and `amp-iso-country-<The ISO code>` e.g. `amp-iso-country-gb` if you're in Great Britain. 

When in GB:
<img width="372" alt="Screen Shot body tag" src="https://user-images.githubusercontent.com/3028997/62608313-8b005b00-b8f7-11e9-817b-848eb6cb4f53.png">

When using a VPN for France: `amp-iso-country-fr`, `amp-geo-group-eea` & visible consent banner
<img width="594" alt="Screen Shot VPN from France" src="https://user-images.githubusercontent.com/3028997/62609016-e8e17280-b8f8-11e9-9b0b-deb36ca16607.png">

When using a VPN to access a country outside of the EEA countries list. e.g. Canada, there is `amp-geo-no-group` instead of `amp-geo-group-eea`. And the consent banner is not shown. 
<img width="593" alt="Screen Shot VPN from Canada" src="https://user-images.githubusercontent.com/3028997/62609014-e848dc00-b8f8-11e9-8d1d-513ad82d687b.png">


The consent banner should only be visible on *AMP pages* if you're in one of these countries: Here are all of the country codes: https://github.com/bbc/psammead/blob/latest/packages/components/psammead-amp-geo/src/index.jsx#L6 


---
